client/build_test.go

```diff
@@ -2086,6 +2087,53 @@ func testClientGatewayNilResult(t *testing.T, sb integration.Sandbox) {
 	require.NoError(t, err)
 }
 
+func testClientGatewayEmptyImageExec(t *testing.T, sb integration.Sandbox) {
+	c, err := New(sb.Context(), sb.Address())
```

Comment:
Needs a skip on direct push with dockerd worker:

```suggestion
	integration.SkipIfDockerd(t, sb, "direct push")
	c, err := New(sb.Context(), sb.Address())
```